eng/pipelines/aggregate-reports.yml

```diff
@@ -16,13 +16,13 @@ jobs:
       displayName: 'Generate Reports'
       inputs:
         mavenPomFile: pom.xml
-        options: '$(DefaultOptions) -Dinclude-non-shipping-modules -Ddependency-checker -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -pl com.azure:jacoco-test-coverage -am'
+        options: '$(DefaultOptions) -Dinclude-non-shipping-modules -Ddependency-checker -Dgpg.skip -pl com.azure:jacoco-test-coverage -am'
```

Comment:
@JimSuplizio also has another PR to fix the aggregate reports pipeline. 
https://github.com/Azure/azure-sdk-for-java/pull/10086